lib/components/viewers/stop-viewer.js

```diff
@@ -99,17 +147,42 @@ class StopViewer extends Component {
           <div className='header-text'>
             {stopData
               ? <span>{stopData.name}</span>
-              : <span>Loading Stop..</span>
+              : <span>Loading Stop...</span>
             }
+            <Button
+              onClick={this._toggleFavorite}
+              bsSize='large'
+              style={{
+                color: this._isFavorite() ? 'yellow' : 'black',
+                padding: 0,
+                marginLeft: '5px'
+              }}
+              bsStyle='link'>
+              <Icon type={this._isFavorite() ? 'star' : 'star-o'} />
+            </Button>
           </div>
           <div style={{ clear: 'both' }} />
         </div>
 
         {stopData && (
           <div className='stop-viewer-body'>
-            {/* plan trip from/to here buttons. TODO: Can this be combined w/ SetFromToButtons? */}
+            {/*
+              Plan trip from/to here buttons.
+              TODO: Can this be combined w/ SetFromToButtons?
+            */ }
             <div>
-              <div><b>Stop ID</b>: {stopId}</div>
+              <div>
+                <b>Stop ID</b>: {stopId}
+                <button
+                  className='link-button pull-right'
+                  style={{ fontSize: 'small' }}
+                  onClick={this._refreshStopTimes}>
+                  <Icon
+                    className={spin ? 'fa-spin' : ''}
+                    type='refresh' />{' '}
+                  {moment(stopData.stopTimesLastUpdated).format('h:mm:ss a')}
```

Comment:
There is a config.yml entry for `datetime.timeFormat`. Perhaps we should use that here?